commit fffab588a4c3edb7081e916b4c5653263d7cf7cb (refs/changes/91/1691/4)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2017-03-31T09:23:11+00:00 (2 years, 6 months ago)
    
    PAPI-136: Add a test to cover multiple wildcards
    Reviewed by: Pedro P. Candel <pedro@joyent.com>

diff --git a/test/api.test.js b/test/api.test.js
index 802f5ed..522e5c0 100644
--- a/test/api.test.js
+++ b/test/api.test.js
@@ -584,6 +584,18 @@ test('GET /packages (Search by wildcard)', function (t) {
 
 
 
+test('GET /packages (Search by multiple wildcard)', function (t) {
+    var query = '/packages?name=*pi_test_*';
+
+    var testFilter = function (p) {
+        return /^api_test_/.test(p.name);
+    };
+
+    searchAndCheckPkgs(t, query, testFilter);
+});
+
+
+
 test('GET /packages (Search by multiple fields)', function (t) {
     var query = '/packages?name=api_test_*&owner_uuids=' + uuid();
 
